portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -31,32 +33,45 @@
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
+  //initialize the number of comments here so that the system always remembers even after refreshing with default 5
+  int numComments;
+
+  /**
+   * This method receives a client's request for their comments data and responds with a json with the data
+   * @param request This holds the client's HttpServletRequest information for the number of comments to display
+   * @param response This holds the server's HttpServletResponse to the client's request
+   */
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-      
-    int numComments;
+    
+    // get the number of MAX comments to display
     try {
       numComments = Integer.parseInt(request.getParameter("numComments"));
-    } catch(NumberFormatException e) {
-      // Default number of comments
+    } catch (NumberFormatException e) {
       numComments = 5;
```

Comment:
for easy readability, can we set this "5" to a global variable and reference it here? 
That way, if you need to find the defaultNumComments elsewhere, you can reference it easily